pagerduty/event_orchestration_path_util.go

```diff
@@ -223,3 +224,20 @@ func flattenEventOrchestrationPathExtractions(e []*pagerduty.EventOrchestrationP
 	}
 	return res
 }
+
+func convertEventOrchestrationPathWarningsToDiagnostics(warnings []*pagerduty.EventOrchestrationPathWarning, diags diag.Diagnostics) diag.Diagnostics {
+	if warnings == nil {
+		return diags
+	}
+
+	for _, warning := range warnings {
```

Comment:
Go noob: what is the `_` in `for _, warning :=`

Reply:
Googled and found it was the index 🤦‍♀️

Reply:
Yup! it's an index if you iterate through an array and a key if you iterate through a map